activation/atxdb.go

```diff
@@ -211,8 +196,13 @@ func (db *ActivationDb) ValidateAtx(atx *types.ActivationTx) error {
 		}
```

Comment:
Just to make sure, currently IsGenesis() returns true if the epoch number is either 0 or 1?

Reply:
yes.